diesel/src/pg/types/uuid.rs

```diff
@@ -0,0 +1,52 @@
+extern crate uuid;
+
+use std::io::prelude::*;
+use std::error::Error;
+
+use expression::AsExpression;
+use expression::bound::Bound;
+use pg::Pg;
+use query_source::Queryable;
+use types::{self, ToSql, IsNull, FromSql};
+
+primitive_impls!(Uuid -> (self::uuid::Uuid, pg: (2950, 2951)));
```

Comment:
Total guess here. Is this appropriate? Does there need to be an `expression_impls`, etc too?